R/pkg/R/DataFrame.R

```diff
@@ -392,7 +392,11 @@ setMethod("coltypes",
                 }
 
                 if (is.null(type)) {
-                  stop(paste("Unsupported data type: ", x))
+                  specialtype <- specialtypeshandle(x)
```

Comment:
or rather, `numeric` is better since it matches the mapping in `PRIMITIVE_TYPES`?